### PR TITLE
Hemiplegic quirk

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

Like paraplegic, but instead of a horizontal half of your body being fucked, it's a vertical half!
image
image

## Why It's Good For The Game

Character customization and pure, undistilled human suffering.

## Changelog
:cl:
add: Added Hemiplegic quirk.
/:cl:

### CODE REVIEW

**Comment by AtomTheProphet** [comment]
Chungus, I'd like to know something. What went through your brain when you decided to make this?

File: code/modules/reagents/reagent_containers/cups/drinkingglass.dm [hunk 1]
```diff
@@ -100,3 +100,6 @@
 /obj/item/reagent_containers/cup/glass/drinkingglass/filled/nuka_cola
 	name = "Nuka Cola"
 	list_reagents = list(/datum/reagent/consumable/nuka_cola = 50)
+
+/obj/item/reagent_containers/cup/glass/drinkingglass/filled/half_full
+	list_reagents = list(/datum/reagent/water = 25)
```
Comment (carlarctg) [inline code/modules/reagents/reagent_containers/cups/drinkingglass.dm:105]:
```suggestion
/obj/item/reagent_containers/cup/glass/drinkingglass/filled/half_full
	desc  = "It's a glass of water. It seems half full. Or is it half empty? You're pretty sure it's full of shit."
	list_reagents = list(/datum/reagent/water = 25)
```

**Comment by ChungusGamer666** [comment]
> Chungus, I'd like to know something. What went through your brain when you decided to make this?

A stroke

**Comment by CRITAWAKETS** [comment]
> A stroke

to be fair, that IS the most common way of getting hemiplegia

**Comment by jlsnow301** [comment]
hemiplegic added to the list of words, 'looks like a typo but really isn't'

**Comment by ChungusGamer666** [comment]
> hemiplegic added to the list of words, 'looks like a typo but really isn't'

Someone needs to give lobectomy company after all

**Comment by san7890** [comment]
oh goddammit i forgot to tax you six gbp

**Pull request merged**